src/NopePrimitive.ts

```diff
@@ -61,6 +61,26 @@ abstract class NopePrimitive<T> implements IValidatable<T> {
     return this.test(rule);
   }
 
+  public notOneOf(options: Array<T | NopeReference | Nil>, message = 'Invalid Option') {
+    const resolveNopeRef = (option: T | NopeReference | Nil, context?: { [key: string]: any}) => {
```

Comment:
Would it be possible to extract this to the ./utils.ts file for both `oneOf` and `notOneOf` pretty please? 😄